homeassistant/components/wink/__init__.py

```diff
@@ -25,7 +25,11 @@
 from homeassistant.helpers.event import track_time_interval
 from homeassistant.util.json import load_json, save_json
 
-REQUIREMENTS = ['python-wink==1.10.1', 'pubnubsub-handler==1.0.2']
+REQUIREMENTS = [
+    'python-wink==1.10.1',
+    'pubnubsub-handler==1.0.2',
+    'pubnub==4.1.2'
```

Comment:
Adding `pubnub` is not needed as it's already a [dependency](https://github.com/w1ll1am23/pubnubsub-handler/blob/master/setup.py#L14) of `pubnubsub-handler`.

Thus, `pubnubsub-handler` should update it and probably remove the pinning but this is @w1ll1am23's call.

Reply:
Totally true, but I felt it would make sense to have it here considering the library wasn't updated in 2 years.
I'll wait to see what @w1ll1am23 prefer :)

Reply:
I am all for updating pubnubsub-handler to the newest version. @fabaff are you suggesting removing pubnub from pubnubsub-handler to allow HA to maintain the version?

Reply:
It depends. If `pubnubsub-handler` needs `pubnub` then it should stay there. If it's only needed for the integration then we should handle it.

Remove the release pinning in `pubnubsub-handler` should definitely considered because pinning releases in modules doesn't play nice with other modules.